react/react1/week1/todo-list/src/componate/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import TodoList from './TodoList';
+import Header from './Header';
+import Footer from './Footer';
+import todoList from '../Todo';
+
+function createTodo(todoList) {
+    return (
+      <TodoList
+        key={todoList.id}
+        description={todoList.description}
+        date={todoList.date}
+      />
+    );
+  }
+function App(){
+return  <div>
+<Header />
```

Comment:
try not to create components without reason, we mostly create components in React when it's going to be reusable.

Reply:
I have tried to make separate header, footer and main section.